### ISSUE TITLE
All commands fail if a user's display name doesn't match their real username

### ISSUE THREAD

**Issue by vivlim** [Macuyiko/minecraft-python#27]
If a user's display name is anything other than their username, minecraft-python will fail to run any commands since it tries to route messages to the user's display name.

This was happening to me because the EssentialsX plugin was configured to prefix op usernames with a red color code, causing `MinecraftPyServerPlugin.send()` to fail.

I think this could be fixed by replacing all instances of `player.getDisplayName` with `player.getName`, but I haven't tried it (yet)

```
[01:19:41] [Server thread/ERROR]: null
org.bukkit.command.CommandException: Unhandled exception executing command 'py' in plugin MinecraftPyServer v0.1
	at org.bukkit.command.PluginCommand.execute(PluginCommand.java:47) ~[patched_1.14.4.jar:git-Paper-175]
	at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:159) ~[patched_1.14.4.jar:git-Paper-175]
	at org.bukkit.craftbukkit.v1_14_R1.CraftServer.dispatchCommand(CraftServer.java:739) ~[patched_1.14.4.jar:git-Paper-175]
	at net.minecraft.server.v1_14_R1.PlayerConnection.handleCommand(PlayerConnection.java:1800) ~[patched_1.14.4.jar:git-Paper-175]
	at net.minecraft.server.v1_14_R1.PlayerConnection.a(PlayerConnection.java:1608) ~[patched_1.14.4.jar:git-Paper-175]
	at net.minecraft.server.v1_14_R1.PacketPlayInChat.a(PacketPlayInChat.java:47) ~[patched_1.14.4.jar:git-Paper-175]
	at net.minecraft.server.v1_14_R1.PacketPlayInChat.a(PacketPlayInChat.java:5) ~[patched_1.14.4.jar:git-Paper-175]
	at net.minecraft.server.v1_14_R1.PlayerConnectionUtils.lambda$ensureMainThread$0(PlayerConnectionUtils.java:23) ~[patched_1.14.4.jar:git-Paper-175]
	at net.minecraft.server.v1_14_R1.TickTask.run(SourceFile:18) ~[patched_1.14.4.jar:git-Paper-175]
	at net.minecraft.server.v1_14_R1.IAsyncTaskHandler.executeTask(IAsyncTaskHandler.java:136) ~[patched_1.14.4.jar:git-Paper-175]
	at net.minecraft.server.v1_14_R1.IAsyncTaskHandlerReentrant.executeTask(SourceFile:23) ~[patched_1.14.4.jar:git-Paper-175]
	at net.minecraft.server.v1_14_R1.IAsyncTaskHandler.executeNext(IAsyncTaskHandler.java:109) ~[patched_1.14.4.jar:git-Paper-175]
	at net.minecraft.server.v1_14_R1.MinecraftServer.aX(MinecraftServer.java:1013) ~[patched_1.14.4.jar:git-Paper-175]
	at net.minecraft.server.v1_14_R1.MinecraftServer.executeNext(MinecraftServer.java:1006) ~[patched_1.14.4.jar:git-Paper-175]
	at net.minecraft.server.v1_14_R1.IAsyncTaskHandler.awaitTasks(IAsyncTaskHandler.java:119) ~[patched_1.14.4.jar:git-Paper-175]
	at net.minecraft.server.v1_14_R1.MinecraftServer.sleepForTick(MinecraftServer.java:990) ~[patched_1.14.4.jar:git-Paper-175]
	at net.minecraft.server.v1_14_R1.MinecraftServer.run(MinecraftServer.java:923) ~[patched_1.14.4.jar:git-Paper-175]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_212]
Caused by: java.lang.NullPointerException
	at com.macuyiko.minecraftpyserver.MinecraftPyServerPlugin.send(MinecraftPyServerPlugin.java:74) ~[?:?]
	at com.macuyiko.minecraftpyserver.jython.JyCommandExecutor.onCommand(JyCommandExecutor.java:38) ~[?:?]
	at org.bukkit.command.PluginCommand.execute(PluginCommand.java:45) ~[patched_1.14.4.jar:git-Paper-175]
	... 17 more
```

**Comment by Macuyiko** [comment]
Thanks for reporting this, I'll fix this soon.

**Comment by Tiebe** [comment]
@Macuyiko Have you fixed this yet?

**Comment by Macuyiko** [comment]
Fixed with commit 4021805edff8b51109988104bd53e66e856b90bb
